bad_package/bad.go

```diff
@@ -0,0 +1,4 @@
+
+package bad_package
+
```

Comment:
@codelingobot capture blank lines make me sad

Reply:
@codelingobot capture how about this time?

Reply:
@stagingcodelingobot capture how about you staging?

Reply:
@stagingcodelingobot review

Reply:
@CodeLingoBot capture blank lines make ben sad

(testing in context to see if it repros here, because it worked for me earlier)